### PR TITLE
gopass-git-credentials: rename to git-credential-gopass

### PR DESCRIPTION
**Pull request by sgn**
With this rename, we can use "gopass" as a git-credential,
no need to have the hackery of invoke another shell.

### CODE REVIEW

File: cmd/git-credential-gopass/git-credential.go [hunk 1]
```diff
@@ -243,7 +243,7 @@ func (s *gc) Configure(c *cli.Context) error {
 	if flags == 0 {
 		log.Println("No target given, assuming --global.")
 	}
-	cmd := exec.CommandContext(ctx, "git", "config", flag, "credential.helper", `"!gopass-git-credentials $@"`)
+	cmd := exec.CommandContext(ctx, "git", "config", flag, "credential.helper", "gopass")
```
Comment (dominikschulz) [inline cmd/git-credential-gopass/git-credential.go:246]: Doesn't this configure the `gopass` binary itself as the git credential helper?

**Comment by sgn** [comment]
On 2020-12-31 12:54:46-0800, Dominik Schulz <notifications@github.com> wrote:
> @dominikschulz commented on this pull request.
> 
> 
> 
> > @@ -243,7 +243,7 @@ func (s *gc) Configure(c *cli.Context) error {
>  	if flags == 0 {
>  		log.Println("No target given, assuming --global.")
>  	}
> -	cmd := exec.CommandContext(ctx, "git", "config", flag, "credential.helper", `"!gopass-git-credentials $@"`)
> +	cmd := exec.CommandContext(ctx, "git", "config", flag, "credential.helper", "gopass")
> 
> Doesn't this configure the `gopass` binary itself as the git credential helper?

This configure "git-credential-gopass" as the git credential helper.

https://git-scm.com/docs/gitcredentials#Documentation/gitcredentials.txt-helper

-- 
Danh

**Review by dominikschulz — APPROVED** [review]
LGTM

**Comment by dominikschulz** [comment]
Thank you. Please rebase and add `RELEASE_NOTES=` tags to your commit message(s).

**Pull request closed without merging**